internal/storage/caching/fast_stat_bucket_test.go

```diff
@@ -135,6 +135,80 @@ func (t *CreateObjectTest) WrappedSucceeds() {
 	ExpectEq(obj, o)
 }
 
+////////////////////////////////////////////////////////////////////////
+// CreateChunkUploader
+////////////////////////////////////////////////////////////////////////
+
+type CreateChunkUploaderTest struct {
+	fastStatBucketTest
+}
+
+func init() { RegisterTestSuite(&CreateChunkUploaderTest{}) }
+
+func (t *CreateChunkUploaderTest) CallsEraseAndWrapped() {
+	const name = "taco"
+
+	// Erase
+	ExpectCall(t.cache, "Erase")(name)
+
+	// Wrapped
+	var wrappedReq *gcs.CreateObjectRequest
+	ExpectCall(t.wrapped, "CreateChunkUploader")(Any(), Any(), Any(), Any()).
+		WillOnce(DoAll(SaveArg(1, &wrappedReq), Return(nil, errors.New(""))))
+
+	// Call
+	req := &gcs.CreateObjectRequest{
+		Name: name,
+	}
+
+	_, _ = t.bucket.CreateChunkUploader(context.TODO(), req, 1, nil)
+
+	AssertNe(nil, wrappedReq)
+	ExpectEq(req, wrappedReq)
+}
+
+func (t *CreateChunkUploaderTest) WrappedFails() {
+	var err error
+
+	// Erase
+	ExpectCall(t.cache, "Erase")(Any())
+
+	// Wrapped
+	ExpectCall(t.wrapped, "CreateChunkUploader")(Any(), Any(), Any(), Any()).
+		WillOnce(Return(nil, errors.New("taco")))
```

Comment:
nit: put "taco" in a constant inside method similar to CallsEraseAndWrapped

Reply:
Done.